common/api/core-frontend.api.md

```diff
@@ -8115,11 +8146,24 @@ export namespace RealityDataSource {
     export function createOrbitGtBlobPropsFromKey(rdSourceKey: RealityDataSourceKey): OrbitGtBlobProps | undefined;
     // @alpha
     export function fromKey(key: RealityDataSourceKey, iTwinId: GuidString | undefined): Promise<RealityDataSource | undefined>;
+    // @alpha
+    export function setBaseUrl(id: string): void;
+}
+
+// @alpha
+export class RealityDataSourceGP3DTProvider implements RealityDataSourceProvider {
+    constructor(options: RealityDataSourceGP3DTProviderOptions);

```

Comment:
Am I missing it, or did RealityDataSourceGP3DTProviderOptions not show up in extract-api?

Reply:
You're right. pretty sure that's because I forgot to export it, oops